spec/registry.md

```diff
@@ -417,6 +455,24 @@ All other values produce a _Bad Operand_ error.
 > or the type `com.ibm.icu.util.CurrencyAmount` can be used to set the currency and related
 > options (such as the number of fraction digits).
 
+### Currency Operands
```

Comment:
Having this section here seems odd. It would be better to have this text directly included in the "`:currency` Operands" section.